lib/src/main/java/com/diffplug/spotless/FeatureClassLoader.java

```diff
@@ -74,4 +76,26 @@
 		return super.findClass(name);
 	}
 
+	/**
+	 * Making spotless Java 9+ compatible. In Java 8 (and minor) the bootstrap
+	 * class loader saw every platform class. In Java 9+ it was changed so the
+	 * bootstrap class loader does not see all classes anymore. This might lead
+	 * to ClassNotFoundException in formatters (e.g. freshmark).
+	 *
+	 * @return <code>null</code> on Java 8 (and minor), otherwise <code>PlatformClassLoader</code>
+	 */
+	@Nullable
+	private static ClassLoader getParentClassLoader() {
+		if (JavaVersion.getMajorVersion() >= 9) {
+			try {
+				return (ClassLoader) ClassLoader.class.getMethod("getPlatformClassLoader").invoke(null);
```

Comment:
Small nit, but you never have to catch and rethrow `RuntimeException` - that is effectively a no-op.  We use the following pattern [throughout the project](https://github.com/diffplug/spotless/search?utf8=%E2%9C%93&q=asRuntime&type=):

```java
} catch (Exception e) {
    throw Errors.asRuntime(e);
}
```

Reply:
Errors.asRuntime comes from durian-core which is in lib just available as testCompile and per comment lib must not have dependencies:

> // zero runtime reqs is a hard requirements for spotless-lib

So I kind of replicated the behavior of Errors.asRuntime.

Reply:
Great catch!  Should have said `ThrowingEx.asRuntime`

Reply:
https://github.com/diffplug/spotless/blob/master/lib/src/main/java/com/diffplug/spotless/ThrowingEx.java

Reply:
Got it. Will change it.